добавил страницу toys

Grishan4ik opened the pull request.

Grishan4ik committed:
добавил страницу toys

Grishan4ik committed:
added pages : blog and contact us

MitrofanovaO commented on index.html:
```diff
@@ -2,23 +2,30 @@
 <html lang="en">
```
проверку на валидацию страница не прошла, проверь
![image](https://user-images.githubusercontent.com/24611929/119888913-fee57a80-bf46-11eb-98dd-3b7277f3230a.png)

MitrofanovaO commented on contact us.html:
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```
не называй страницы через пробел, если нужно назвать в 2 слова - ставь тире 
![image](https://user-images.githubusercontent.com/24611929/119889446-9ba81800-bf47-11eb-8762-e4de26f3453b.png)

MitrofanovaO commented on contact us.html:
```diff
@@ -0,0 +1,57 @@
+        <a href="index.html"><img src="image/logo.jpg" alt="logo"></a>
+        <nav>
+            <ul>
+                <li><a href="#">Blog</a></li>
+                <li><a href="catalog.html">Catalog</a></li>
+                <li><a href="#">About Us</a></li>
```
у тебя добавились все страницы пропиши везде пути, чтобы можно было свободно по сайту передвигаться, но перед этим надо отредактировать структуру. у тебя должна быть папка pages и там все страницы кроме главной index

MitrofanovaO commented on blog.html:
```diff
@@ -0,0 +1,146 @@
+    <div class="comapany_rating">
+        <p>Company Rating</p>
+        
+        <img src="image/star.png" alt="Star">
```
картинка не отображается, в папке она с большой буквы, надо это исправить, все картинки должны быть с маленькой буквы и если ты придерживаешься разделения слов  в названии картинок постановкой тире, то это должно быть во всех картинках где есть несколько слов

MitrofanovaO commented on blog.html:
```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
надо бы добавить на все страницы meta name="viewport" для того чтобы сайт был адаптивным, ну по сути тебе сейчас это не понадобится, но всё же, это очень важная строчка, почитай про это

MitrofanovaO commented on blog.html:
```diff
@@ -0,0 +1,146 @@
+    <meta charset="UTF-8">
+    <title>blog</title>
+</head>
+
```
много лишних пробелов по страницам, как в стилях так и в html

MitrofanovaO commented on blog.html:
```diff
@@ -0,0 +1,146 @@
+        </ul>
+    </nav>
+    
+    <div><a href="#"><img src="image/search.png" alt="seach"></a></div>
+    
+    <div><a href="#">Contact  Us</a></div>
```
этидва блока лучше объеднить в один обернув в див, это тебе в дальнейшем поможет при позиционировании данного блока. я бы лучше сделала его списком ul>li а кнопку поиска сделала button потому что это не ссылка на другую страницу

MitrofanovaO commented on index.html:
```diff
@@ -27,35 +34,33 @@
-            <img src="image/toys.png" alt="pic">
-            <h2>TOYS</h2>
-        </section>
-        <section class="banner">
-            <img src="image/accessories.png" alt="pic">
-            <h2>ACCESSORIES</h2>
+    
+        <section class="pic">
+    <ul class="dots-none">
+        <li><h2><a href="">APPAREL
+                KIDS<img src="image/apparel_kids.png"  alt="pic"></a></h2></li>
+            
+        <li><h2><a href="">HOLIDAY SHOP<img src="image/holiday_shop.png" alt="pic"></a></h2></li>
 
+        <li><h2><a href="">TOYS<img src="image/toys.png" alt="pic"></a></h2></li>
+        
+        <li><h2><a href="">ACCESSORIES<img src="image/accessories.png" alt="pic"></a></h2></li>
+            
+</ul>
         </section>
```
это один отдельный блок с заголовком Category Products (блок категорий), не поняла почему у тебя заголовок это отдельная секция, и дальше у тебя секция уже с категориями
названия категорий может быь и могут быть заголовками, но точно не h2 они слишком значимые для страницы, обычно заголовки ставят там где заголовок у  блока, тут  у тебя он уже есть, если и можно поставить заголовок для категории, то только h3 и ниже

MitrofanovaO commented on index.html:
```diff
@@ -27,35 +34,33 @@
+    
+        <section class="pic">
+    <ul class="dots-none">
+        <li><h2><a href="">APPAREL
+                KIDS<img src="image/apparel_kids.png"  alt="pic"></a></h2></li>
```
в h2 возможно максимум теги div span p a (в отдельных случаях) но никак не картинки, у тебя нет никаких таких случаев когда, не понимаю почему ты в заголовок все поместил, тут у тебя должна быть li в ней ссылка, внутри ссылки картинка и заголовок

Grishan4ik committed:
3

исправил ошибки на которые было указано. проставил пути в ссылках, чтобы переходить по сайту

Grishan4ik committed:
3

исправил ошибки

Grishan4ik committed:
4

corrected the comments

Grishan4ik committed:
added by file common css

Grishan4ik committed:
need help

Grishan4ik committed:
stylized the main page

Grishan4ik committed:
applied css to the catalog page

Grishan4ik committed:
final pull

I'm waiting for comments
